lib/webpacker/railtie.rb

```diff
@@ -12,6 +13,10 @@ class Webpacker::Engine < ::Rails::Engine
       include Webpacker::Helper
     end
 
+    ActiveSupport.on_load :before_eager_load do
+      app.config.x.webpacker[:reload] = false
```

Comment:
instead of using the x namespace what about creating a webpacker namespace? Also you don't need to use `ActiveSupport.on_load :before_eager_load do`. The railtie can set the value in its body.